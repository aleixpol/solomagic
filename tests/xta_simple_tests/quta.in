\ref something0
\mt somethingsomething (qu)ta somethingsomething

\ref something1
\mt somethingsomething (q)uta somethingsomething

