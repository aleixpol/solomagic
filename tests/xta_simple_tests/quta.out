\ref something0
\mt somethingsomething (qu)ta somethingsomething
\ma somethingsomething xta somethingsomething

\ref something1
\mt somethingsomething (q)uta somethingsomething
\ma somethingsomething (q)uta somethingsomething
